src/utils/racingResult.js

```diff
@@ -0,0 +1,12 @@
+import { isOverFour } from './isOverFour.js'
+
+export const racingResult = carsInfo => {
+	for (const car of carsInfo) {
+		const isOverFourBoolean = isOverFour()
+		if (isOverFourBoolean) {
+			carsInfo.set(car[0], (car[1] += 1))
+		}
+	}
+
+	return carsInfo
+}
```

Comment:
`racingResult`라는 함수와 `isOverFour`라는 함수를 분리하신 이유가 있으신가요? 둘다 그리 길지 않고, 어차피 레이싱의 과정이기 때문에 한 함수에서 실행해줘도 될 것 같다고 생각했습니다!

```suggestion
export const racingResult = carsInfo => {
	for (const car of carsInfo) {
		const randomNum = MissionUtils.Random.pickNumberInRange(0, 9);
		const isOverFourBoolean = randomNum >= 4;
		if (isOverFourBoolean) {
			carsInfo.set(car[0], (car[1] += 1));
		}
	}

	return carsInfo;
}
```

Reply:
둘이 코드가 길이는 않지만 제가 생각했을때 최대한 의존성을 줄이기 위해서 저렇게 진행해봤었습니다. 
만약 4이상이 아니라 7이상으로 게임의 규칙이 변경되었다고 했을때 isOverfour에 대한 코드만 보고 변경을 하면 유지보수 측면에서 좀더 편할거라고 생각을 합니다. 레이싱에 대한 게임 로직도 늘고 했을때 저기 있는 코드를 다 봐야한다는 점에서 생각해봤던거같습니다 !